mne/io/edf/edf.py

```diff
@@ -118,15 +118,30 @@ class RawEDF(BaseRaw):
     To retrieve correct event values (bits 1-16), one could do:
 
         >>> events = mne.find_events(...)  # doctest:+SKIP
-        >>> events[:, 2] >>= 8  # doctest:+SKIP
+        >>> events[:, 2] &= (2**16 - 1)  # doctest:+SKIP
```

Comment:
This could probably also be done with `find_events(..., mask=2**16 - 1, mask_type='and')`. But I don't mind showing people the direct math :)